charts/fybrik/values.yaml

```diff
@@ -79,7 +79,9 @@ coordinator:
 
   # Configure the vault instance to be used by the coordinator manager
   vault:
-    # Set to the Vault address. 
+    # if you don't plan to use Vault, set it to false. If it is false, you can remove all other entries below
```

Comment:
I would replace this comment with a strong warning explaining that this could be done only if all deployed modules do not require getting credentials to the data from Fybrik.

Reply:
done